unicodetools/src/test/java/org/unicode/unittest/TestUnicodeSet.java

```diff
@@ -33,11 +33,11 @@ private void checkOrder(String d1, String d2, int expected) {
         assertEquals(
                 d1 + " ?< " + d2,
                 expected,
-                UnicodePropertySymbolTable.DOUBLE_STRING_COMPARATOR.compare(d1, d2));
+                UnicodePropertySymbolTable.VERSION_STRING_COMPARATOR.compare(d1, d2));
         assertEquals(
                 d2 + " ?< " + d1,
                 -expected,
-                UnicodePropertySymbolTable.DOUBLE_STRING_COMPARATOR.compare(d2, d1));
+                UnicodePropertySymbolTable.VERSION_STRING_COMPARATOR.compare(d2, d1));
```

Comment:
assertEquals() is now stricter than the actual return value from the comparator. It expects the return values to be pinned to -1/0/+1 but compareTo() can return larger negative/positive values.

```
TestUnicodeSet.TestAge:16->checkOrder:33->TestFmwkMinusMinus.assertEquals:50 3.1 ?< 3.2 ==> expected: <-1> but was: <-32768>
```

I suggest changing the test code to extract the "sign" of the return value.
Alternatively, you could change the comparator to return only the sign value.